packages/noya-designsystem/src/components/Button.tsx

```diff
@@ -59,11 +61,17 @@ function Button({
   id,
   tooltip,
   active = false,
+  disabled = false,
   onClick,
   children,
 }: ButtonRootProps) {
   const buttonElement = (
-    <ButtonElement id={id} active={active} onClick={onClick}>
+    <ButtonElement
+      id={id}
+      active={active}
+      disabled={disabled}
+      onClick={onClick}
```

Comment:
Can we prevent the `onClick` from firing in the case of a disabled button? (does that happen already?)

Reply:
it happens already, if you see in the html, the button it's disabled. I think the button styled components is already handling it.  
![image](https://user-images.githubusercontent.com/16924758/113801220-02663d80-971e-11eb-8f8f-7859f154568c.png)